tiledb/sm/misc/mgc_dict.cc

```diff
@@ -59,8 +55,10 @@ int magic_mgc_embedded_load(magic_t magic) {
 }
 
 span<const uint8_t> expanded_buffer() {
+  static std::vector<uint8_t> expanded_buffer(magic_mgc_decompressed_size);
+  static std::once_flag once_flag;
   // Thread-safe initialization of the expanded data.
-  static auto expanded_buffer = prepare_data();
+  std::call_once(once_flag, [&]() { prepare_data(expanded_buffer); });
```

Comment:
Now that I see the code, I realize this could be done another way. If `prepare_data` were to return its input argument, it could be used to initialize a second static variable, which would then be the return value.

No need to change it.